### ISSUE TITLE
termux-* commands fail when invoked from within Emacs

### ISSUE THREAD

**Issue by chipschap** [termux/termux-app#608]
I installed the termux-api package and app and all termux-* commands work perfectly at the command line. However, when I launch Emacs and attempt such a command from with Emacs (whether in eshell, from an elisp program, with M-!, etc.) I get either a hang (which fortunately is interruptable with Ctrl-g) or a segfault. An example is termux-open. If, again within Emacs, I launch the command with 'bash termux-open" I get an error 127. A few things work, like termux-info. The difference seems to be anything that eventually invokes "am" in its script, and the "bash termux-open" example in fact points to the line containing "am" in the termux-open script ("am broadcast ...").

I tried to find the point of failure in "am" and I /think/ but cannot be certain or verify that the problem occurs when "am" reads arguments. 

It's hard to see what's different when running inside Emacs except for environment variables. But it's frustrating as the ability to use things like termux-open, termux-clipboard-*, etc., would be really a big benefit.

**Comment by michalbednarski** [comment]
Looks like this is caused by rlimit set by emacs:

```sh
$ unset LD_PRELOAD
$ emacs # am doesn't work

$ cat norlimit.c
int setrlimit() { return 0; }
$ clang norlimit.c -shared -o norlimit.so
$ export LD_PRELOAD=`pwd`/norlimit.so
$ emacs # am works
```

**Comment by chipschap** [comment]
michalbednarski <notifications@github.com> writes:

> $ cat norlimit.c
> int setrlimit() { return 0; }
> $ clang norlimit.c -shared -o norlimit.so
> $ export LD_PRELOAD=`pwd`/norlimit.so
> $ emacs # am works

Many thanks. This works perfectly, and now allows easy use
within Emacs of auctex viewing, openwith functions, etc.... a
huge plus.

Aloha,

-- 
Bob Newell
Honolulu, Hawai`i
* Via Gnus/BBDB/Org/Emacs/Linux *

**Comment by wasco** [comment]
Sorry.
Where is this magic file norlimit.c?
Can't find it.

**Comment by fornwall** [comment]
After merging https://github.com/termux/termux-packages/pull/2239 this should now be fixed by version `25.3-2` of the emacs package, which is just a `pkg up` away.

Nice work in finding this!
